ci/datamgr.py

```diff
@@ -44,22 +128,182 @@ def get_logger(name, level=None, format=None, propagate=False):
 logger = get_logger(Path(__file__).with_suffix('').name)
 
 
-def recreate_database(driver, params, **kwargs):
-    database = params.pop("database", None)
-    url = sa.engine.url.URL(driver, **params)
+def make_ibis_client(env):
```

Comment:
Either rename it to `make_impala_client` or rather put the code into the impala click command.

Reply:
Roger that.

Reply:
Done